pkg/kustomize/kustomize_cli_test.go

```diff
@@ -0,0 +1,63 @@
+package kustomize_test
+
+import (
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"github.com/jenkins-x/jx/pkg/cmd/testhelpers"
+	"github.com/jenkins-x/jx/pkg/kustomize"
+	"github.com/jenkins-x/jx/pkg/log"
+	"github.com/jenkins-x/jx/pkg/util"
+)
+
+func TestKustomizeCLI_FindKustomize_HasKustomize(t *testing.T) {
+	origJXHome, testJXHome, err := testhelpers.CreateTestJxHomeDir()
+	if err != nil {
+		t.Errorf("failed to create a test JX Home directory")
+	}
+
+	defer func() {
+		err := testhelpers.CleanupTestJxHomeDir(origJXHome, testJXHome)
+		if err != nil {
+			log.Logger().Warnf("unable to remove temporary directory %s: %s", testJXHome, err)
+		}
+	}()
+
+	err = os.MkdirAll(filepath.Join(testJXHome, "bin/base/charts"), os.ModePerm)
```

Comment:
I'd not use _bin_ as part of the test directory structure. Use _myApp_ or something. Readers of the test might otherwise get confused.